File: TreasurersApp/ClientApp/src/app/models/ContributionTypeViewModel.ts
export class ContributionTypeViewModel {
  id: number = null;
  contributionCategoryId: number = null;
  contributionCategoryDescription: string;
  contributionTypeName: string = null;
  description: string = null;
  displayOrder: number = null;
  active = false;

  constructor(id: number,
    contributionCategoryId: number,
    contributionCategoryDescription: string,
    contributionTypeName: string,
    description: string,
    displayOrder: number,
    active: boolean) {
    this.id = id;
    this.contributionCategoryId = contributionCategoryId;
    this.contributionCategoryDescription = contributionCategoryDescription;
    this.contributionTypeName = contributionTypeName;
    this.description = description;
    this.displayOrder = displayOrder;
    this.active = active;
  }
}

File: TreasurersApp/Models/ContributionTypeViewModel.cs
using Newtonsoft.Json;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace TreasurersApp.Models
{
    public class ContributionTypeViewModel
    {
        [JsonProperty("id")]
        public int ContributionTypeID { get; set; }

        [JsonProperty("contributionCategoryId")]
        public int CategoryID { get; set; }

        [JsonProperty("contributionCategoryDescription")]
        public string CategoryDescription { get; set; }

        [JsonProperty("contributionTypeName")]
        public string ContributionTypeName { get; set; }

        [JsonProperty("description")]
        public string Description { get; set; }

        [JsonProperty("displayOrder")]
        public int DisplayOrder { get; set; }

        [JsonProperty("active")]
        public bool Active { get; set; }
    }
}
